stop.go

```diff
@@ -8,8 +8,8 @@ import (
 // Stop is a place where vehicles frequently stop.
 type Stop struct {
 	ID        int64     `json:"id"`
-	Latitude  float64   `json:"latitude"`
-	Longitude float64   `json:"longitude"`
+	Latitude  float64   `json:"latitude,string"`
```

Comment:
This shouldn't have to be changed on the backend, the types on the frontend should match those on the backend